runhouse/resources/hardware/cluster.py

```diff
@@ -258,11 +258,26 @@ def delete_configs(self):
 
         super().delete_configs()
 
+    def _should_save_creds(self, folder: str = None) -> bool:
+        """Checks whether to save the creds associated with the cluster.
+        Only do so as part of the save() if the user making the call is the creator"""
+
+        from runhouse.globals import configs
```

Comment:
we already import globals at the top of the file, so can just add `configs` to it?